modules/calc/src/main/java/com/opengamma/strata/calc/CalculationRules.java

```diff
@@ -36,45 +37,73 @@
  *   <li>The market data rules which specify which items of market data should be used for a calculation,
  *   for example which curve group should be used as the source of curves and which market data feed should
  *   be used to request observable values</li>
- *   <li>The reporting rules define how the results should be reported, including the reporting currency</li>
+ *   <li>The reporting currency defines the currency of the result</li>
  * </ul>
  */
 @BeanDefinition
 public final class CalculationRules implements ImmutableBean {
 
-  /** The rules defining how calculations should be performed. */
+  /**
+   * The rules defining how calculations should be performed.
+   * <p>
+   * This controls which {@link CalculationFunction} is used for a calculation.
+   * This defaults to an empty instance if using the builder.
+   */
   @PropertyDefinition(validate = "notNull")
   private final PricingRules pricingRules;
-
-  /** The rules defining what market data should be used in each calculation. */
+  /**
+   * The rules that define what market data should be used in each calculation.
+   * <p>
+   * This controls which view of the market data should be used for a calculation.
```

Comment:
I don't think "...view of the market data..." is obvious enough without more context and I don't think it adds anything more than the previous sentence.

It might be worth elaborating; explaining that there is a global set of market data and the rules choose the subset which is used for each calculation. Or you could just get rid of that sentence.